pkg/ssh/exec.go

```diff
@@ -159,6 +161,23 @@ func Exec(ctx context.Context, iface string, remotePort int, tty bool, inR io.Re
 		}
 	}()
 
+	resize := make(chan os.Signal)
+	signal.Notify(resize, syscall.SIGWINCH)
```

Comment:
Needs to be compiled on windows. Check https://github.com/docker/cli/blob/c76cbdc6a1ba8fead63b5673728d3c1b02cb510b/cli/command/container/tty.go#L71